### DISCUSSION TITLE
表格的处理办法

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#643] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/113572850/197123974-0cefd8a2-81b5-44ee-a9ff-d06c65dc366d.png)
这种情况，咱们是采取怎样的处理办法？

**Comment by PirateBook1** [comment]
@banned-historical-archives

**Reply by PirateBook1** [reply]
我觉得可以标明这里存在表格，然后读者自行点按钮查看原文稿阅读表格。

**Reply by banned-historical-archives** [reply]
表格和插图都不太好处理，我觉得暂时把表格转成图片比较好
新增两种段落类型（插图，插图-表格），填写相对路径就可以显示图片，插图文件放在在图片集目录下
你看怎么样？

**Reply by PirateBook1** [reply]
怎么存放，总不能录入者手动截图然后 push 上来保存吧

**Reply by banned-historical-archives** [reply]
好像没有别的办法

**Reply by PirateBook1** [reply]
我觉得直接标明就好了，可能你可以加一个表格-插图类型，只可以填一个数字，表示这里有一个表格，在 PDF 第几页。没有必要去特意截图

**Reply by banned-historical-archives** [reply]
恩恩，你先用paragraph类型表示吧

**Comment by PirateBook1** [comment]
@banned-historical-archives 这里也修复一下吧

**Reply by PirateBook1** [reply]
最好是一个超链接，能 tag 到具体图片集/PDF 的第几页

**Reply by banned-historical-archives** [reply]
这好像不太通用，如果跨过2页呢

**Reply by PirateBook1** [reply]
不打紧，tag 到最初出现的那页

**Reply by banned-historical-archives** [reply]
pdf不能跳转到指定页数

**Reply by PirateBook1** [reply]
https://github.com/banned-historical-archives/banned-historical-archives.github.io/discussions/643#discussioncomment-3931034

@banned-historical-archives 不太好办，想来想去还是你之前提的插图方式好

**Reply by PirateBook1** [reply]
是不是可以自动 issue 上传，然后名字也用随机 id，可以跟自动化校对配合

**Reply by banned-historical-archives** [reply]
可以

**Comment by banned-historical-archives** [comment, marked as answer]
https://github.com/banned-historical-archives/banned-historical-archives.github.io/issues/701

表格作为插图好还是用类似markdown的格式存表格数据比较好？

**Reply by PirateBook1** [reply]
一律使用截图代替吧，不要再增添工作量了，左边前端直接展示出来
